src/main/java/org/terasology/ligthandshadow/logic/CardSystem.java

```diff
@@ -1,148 +0,0 @@
-/*
- * Copyright 2017 MovingBlocks
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.terasology.ligthandshadow.logic;
-
-import org.joml.Vector3f;
-import org.joml.Vector3i;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.terasology.engine.audio.AudioManager;
-import org.terasology.engine.entitySystem.entity.EntityBuilder;
-import org.terasology.engine.entitySystem.entity.EntityManager;
-import org.terasology.engine.entitySystem.entity.EntityRef;
-import org.terasology.engine.entitySystem.event.ReceiveEvent;
-import org.terasology.engine.entitySystem.prefab.PrefabManager;
-import org.terasology.engine.entitySystem.systems.BaseComponentSystem;
-import org.terasology.engine.entitySystem.systems.RegisterSystem;
-import org.terasology.engine.logic.common.ActivateEvent;
-import org.terasology.module.health.events.OnDamagedEvent;
-import org.terasology.engine.logic.inventory.ItemComponent;
-import org.terasology.engine.logic.location.LocationComponent;
-import org.terasology.engine.math.Side;
-import org.terasology.engine.particles.components.ParticleEmitterComponent;
-import org.terasology.engine.registry.In;
-import org.terasology.engine.rendering.logic.MeshComponent;
-import org.terasology.engine.utilities.Assets;
-import org.terasology.engine.world.BlockEntityRegistry;
-import org.terasology.engine.world.WorldProvider;
-import org.terasology.engine.world.block.Block;
-import org.terasology.engine.world.block.BlockComponent;
-import org.terasology.engine.world.block.BlockManager;
-import org.terasology.engine.world.block.BlockRegion;
-import org.terasology.engine.world.block.family.BlockPlacementData;
-import org.terasology.engine.world.block.regions.BlockRegionComponent;
-
-@RegisterSystem
-public class CardSystem extends BaseComponentSystem {
```

Comment:
Looking at this code, I'm not even sure anymore whether the cards act as doors or not :thinking:

Reply:
P.S. we just found out that our door placement itself is broken right now :eyes: :door: :question:

Reply:
So in that case would it be best to remove this for the time being and try and fix card placement to use as walls as a part of a later PR?

Reply:
I'd vote for keeping it (potentially considering to move it to LASR too) and writing an issue tracking to replace this leveraging `MultiBlock` instead.

Reply:
In that case should I retain CardComponent too.

Reply:
yes, but move it to LASR to resolve the dependency being the wrong way round